app/seeds/review_seeder.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewSeeder
+  REVIEWS = [
+    {
+      rider_id: 1,
+      trip_id: 1,
+      rating: 2,
+      review_body: "Liked it a lot",
+      driver_id: 2
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.